### PR TITLE
Feature/Update readme

### PR DESCRIPTION
**Pull request by amychen1776**
I just added some additional text to clarify on the readme. Will likely have more input after I work through the steps!

### CODE REVIEW

**Comment by amychen1776** [comment]
@matt-winkler Thoughts on this PR?

Update: I see the merge conflict now. Nvm

File: README.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # airflow-fivetran-dbt
-Example orchestration pipeline for Fivetran + dbt managed by Airflow
+The purpose of this github repo is to provide an example how an orchestration pipeline for Fivetran + dbt managed by Airflow would look like. If you have any questions about this, feel free to ask in our [dbt Slack community](https://community.getdbt.com/).
```
Comment (matt-winkler) [inline README.md]: Can we replace the added text with "The purpose of this github repo is to provide an example of an orchestration pipeline for Fivetran + dbt managed by Airflow. If you have any questions about this, feel free to ask in our [dbt Slack community](https://community.getdbt.com/)."

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+This is one way to orchetstrate dbt in coordination with other tools, such as Fivetran for data loading. In this example, our focus is on coordinating a Fivetran sync for loading data to a warehouse, and then triggering a dbt run in an event-driven pipeline. We use the Fivetran and dbt Cloud APIs to accomplish this, with Airflow managing the scheduling / orchestration of the job flow. The final step extracts the `manifest.json` from the dbt run results to capture relevant metadata for downstream logging, alerting and analysis. The code provided in this repository are intended as a demonstration to build upon and should not be utilized as a production-ready solution.
```
Comment (matt-winkler) [inline README.md:6]: Replace the last sentence with "The code provided in this repository is intended as a demonstration to build upon and should not be utilized as a production-ready solution."

File: README.md [hunk 3]
```diff
@@ -19,13 +19,14 @@ This is one way to orchetstrate dbt in coordination with other tools, such as Fi
...
-- configurable approach which can be extended to handle additional Fivetran connectors and dbt job definitions  
-- captures relevant data from a job run which could be shipped to downstream logging & analytics services. It would also be feasible to log interim job status data using this setup, though we did not build it  into the current python codebase
+## Highlights 
+This repository aims to: 
+- display logical isolation of data load (Fivetran), data transform (dbt) and orchestration (Airflow) functions
+- provide Airflow code would be portable to a tool like [Astronomer](https://www.astronomer.io/)  
+- avoids complexity of re-creating dbt DAG in Airflow 
+- demonstrate orchestrating Fivetran and dbt in an event-driven pipeline  
+- provide a configurable approach which can be extended to handle additional Fivetran connectors and dbt job definitions  
+- capture relevant data from a job run which could be shipped to downstream logging & analytics services. It would also be feasible to log interim job status data using this setup, though we did not build it  into the current python codebase
```
Comment (matt-winkler) [inline README.md]: One extra space after "thought we did not built it  " should be removed

**Review by matt-winkler — CHANGES_REQUESTED** [review]
A few nit-picky grammar things but otherwise LGTM, thanks for the prompt!

**Pull request merged**